tests/FakeItEasy.Specs/ThenSpecs.cs

```diff
@@ -17,6 +17,43 @@ public interface IFoo
             int Baz();
         }
 
+        [Scenario]
+        public static void VoidMethodWithInvokesAndDoesNothing(

```

Comment:
I'd originally dropped all cases of `Invokes(…).DoesNothing()` in the tests, but realized we could then accidentally remove the possibility from the API, which we could do, if you like, but I'd rather do consciously and remove the last tests then.

Reply:
Sounds good to me